src/administration/integrations/gitlab.md

```diff
@@ -0,0 +1,69 @@
+# GitLab
+
+The [GitLab](https://gitlab.com) integration allows you to manage your
+Platform.sh environments directly from your GitLab repository.
+
+Features supported:
+
+* Create a new environment when creating a branch or opening a
+    pull request on GitLab.
+* Rebuild the environment when pushing new code to GitLab.
+* Delete the environment when merging a pull request.
+
+## Setup
+
+### 1. Generate a token
+
+To integrate your Platform.sh project with an existing GitLab
+repository, you first need to generate a token on your GitLab user
+profile. Simply go to your Settings page on GitLab and click
+`Access Tokens`.
+
+Fill the `Name` field for example with "Platform.sh Integration" and optionally
+set an expiration time.
```

Comment:
Here and above should use soft word wrapping.  (Probably a copy-paste from another page that we haven't gotten around to fixing yet.)